<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14993" started-at="2016-10-10T13:36:11Z" finished-at="2016-10-10T13:36:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14993" started-at="2016-10-10T13:36:11Z" finished-at="2016-10-10T13:36:26Z">
      <class name="com.pack.MavenJunkinsTestCase">
        <test-method status="PASS" signature="openGoogle()[pri:0, instance:com.pack.MavenJunkinsTestCase@23ceabc1]" name="openGoogle" is-config="true" duration-ms="13560" started-at="2016-10-10T19:06:11Z" finished-at="2016-10-10T19:06:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGoogle -->
        <test-method status="FAIL" signature="enterText()[pri:0, instance:com.pack.MavenJunkinsTestCase@23ceabc1]" name="enterText" duration-ms="433" started-at="2016-10-10T19:06:24Z" finished-at="2016-10-10T19:06:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.pack.MavenJunkinsTestCase.enterText(MavenJunkinsTestCase.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterText -->
        <test-method status="PASS" signature="CloseWebDriver()[pri:0, instance:com.pack.MavenJunkinsTestCase@23ceabc1]" name="CloseWebDriver" is-config="true" duration-ms="980" started-at="2016-10-10T19:06:25Z" finished-at="2016-10-10T19:06:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseWebDriver -->
      </class> <!-- com.pack.MavenJunkinsTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
